LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2013-2015
```

Comment:
The git log shows that the first commit to this repository was 2013.  Prior to that, this functionality was part of the git plugin.  I didn't attempt to trace the history of any source files back into the git plugin tree.

I assumes the license file at the root level is intended to represent this plugin and its repository.  However, I'm sure there are better experts than me to decide which copyright date should be applied based on the history of the files (if that is better than history of the repository)